src/action-ros-ci.ts

```diff
@@ -125,32 +136,51 @@ async function run() {
 		const packageNameList = packageName.split(RegExp("\\s"));
 		const rosWorkspaceName = "ros_ws";
 		const rosWorkspaceDir = path.join(workspace, rosWorkspaceName);
-		const sourceRosBinaryInstallation = core.getInput(
-			"source-ros-binary-installation"
-		);
-		const sourceRosBinaryInstallationList = sourceRosBinaryInstallation
-			? sourceRosBinaryInstallation.split(RegExp("\\s"))
-			: [];
-
+		const targetRos1Distro = core.getInput("target-ros1-distro");
+		const targetRos2Distro = core.getInput("target-ros2-distro");
 		const vcsRepoFileUrlListAsString = core.getInput("vcs-repo-file-url") || "";
 		const vcsRepoFileUrlList = vcsRepoFileUrlListAsString.split(RegExp("\\s"));
-		const vcsRepoFileUrlListNonEmpty = vcsRepoFileUrlList.filter(x => x != "");
-		const vcsRepoFileUrlListResolved = vcsRepoFileUrlListNonEmpty.map(x =>
+		const vcsRepoFileUrlListNonEmpty = vcsRepoFileUrlList.filter(
+			(x) => x != ""
+		);
+		const vcsRepoFileUrlListResolved = vcsRepoFileUrlListNonEmpty.map((x) =>
 			resolveVcsRepoFileUrl(x)
 		);
 
 		const coverageIgnorePattern = core.getInput("coverage-ignore-pattern");
 
 		let commandPrefix = "";
-		if (sourceRosBinaryInstallation) {
+		if (targetRos1Distro) {
 			if (process.platform !== "linux") {
```

Comment:
This error block is no longer entirely correct, since we use this argument for `rosdep install` further on.